Cataclysm/APLs/RogueAssasination.simc

```diff
@@ -16,32 +21,37 @@ actions+=/call_action_list,name=cds
 actions+=/shiv,if=debuff.dispellable_enrage.up
 actions+=/call_action_list,name=defensives

```

Comment:
This should probably be gated behind a condition that `toggle.defensives` is active to stay aligned with the addon's damage-first design mentality for DPS characters.